examples/re2/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2016-2017, Ruslan Baratov
+# Copyright (c) 2017, David Hirvonen
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-re2)
+
+hunter_add_package(re2)
+find_package(re2 CONFIG REQUIRED)
```

Comment:
Should be `find_package(RE2 CONFIG REQUIRED)`:
```
Could not find a package configuration file provided by "re2" with any of
the following names:

  re2Config.cmake
  re2-config.cmake
```

Reply:
Thanks.  Done.  Test: `SUCCESS`